### PR TITLE
berry: bumped to 0.1.8

### PR DESCRIPTION
**Pull request by lowercasename**
## Installed manifest of package

```
/var/db/kiss/installed/berry/version
/var/db/kiss/installed/berry/sources
/var/db/kiss/installed/berry/manifest
/var/db/kiss/installed/berry/depends
/var/db/kiss/installed/berry/checksums
/var/db/kiss/installed/berry/build
/var/db/kiss/installed/berry/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/local/share/man/man1/berryc.1
/usr/local/share/man/man1/berry.1
/usr/local/share/man/man1/
/usr/local/share/man/
/usr/local/share/
/usr/local/bin/berryc
/usr/local/bin/berry
/usr/local/bin/
/usr/local/
/usr/
```

## Existing package

- [X] I am the maintainer of this package.

### CODE REVIEW

File: community/berry/version [hunk 1]
```diff
@@ -1 +1 @@
-0.1.7 1 
+0.1.8 2
```
Comment (dilyn-corner) [inline community/berry/version:1]: This should just be `0.1.8 1`

**Review by dilyn-corner — CHANGES_REQUESTED** [review]
`berry` seems all kinds of wonky, build-system wise. The `configure` script doesn't even have the correct version set!

Instead of fiddling with that nonsense, you can just do:
```
make
make DESTDIR="$1" bindir=/usr/bin mandir=/usr/share/man install
```

And that will put everything in the correct places. 

Just do those changes and force push them (`git push --force`) to this branch and you should be fine.

File: community/berry/version (hunk 1 above)
Comment (lowercasename) [reply community/berry/version:1]: Ah, that's me misunderstanding the docs! What does this phrase mean?

> the second field should contain the version number of the repository files themselves

I took it to mean that every time the repository files get updated to a new package version, that number is incremented.

**Comment by lowercasename** [comment]
Thanks for looking over this and for your kind suggestion of the new build script, @dilyn-corner!

File: community/berry/version (hunk 1 above)
Comment (dilyn-corner) [reply community/berry/version]: I see how that can be confusing. 
You can think of the second version field as being the version of those files *at that version*. Some people call this an epoch (or less commonly, age) version. Essentially, this version changes when the package itself has changed in some way that requires a *rebuild*, even though the package itself has not changed (see the [recent changes to alsa-lib](https://github.com/kiss-community/repo/commit/83fde325c2f710dbe933e8b6cb2c74c11ec0a127) for an example).

**Pull request merged**